tests/unit_tests/dagmc/test_model.py

```diff
@@ -0,0 +1,117 @@
+import pkg_resources
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+import openmc
+from openmc import ZPlane, YPlane, XPlane, Cell
+
+pytestmark = pytest.mark.skipif(
+    not openmc.lib._dagmc_enabled(),
+    reason="DAGMC CAD geometry is not enabled.")
+
+def test_model_differentiate_with_DAGMC():
+    PITCH = 1.26
+
+    mats = {}
+    mats["Fuel"] = openmc.Material(1, "Fuel")
+    mats["Fuel"].add_nuclide("U235", 0.03)
+    mats["Fuel"].add_nuclide("U238", 0.97)
+    mats["Fuel"].add_nuclide("O16", 2.0)
+    mats["Fuel"].set_density("g/cm3", 10.0)
+    mats["Fuel"].name = "Fuel"
+
+    mats["Clad"] = openmc.Material(name="zirconium")
+    mats["Clad"].add_element("Zr", 1.0)
+    mats["Clad"].set_density("g/cm3", 6.6)
+    mats["Clad"].name = "Clad"
+
+    mats["Water"] = openmc.Material(name="h2o")
+    mats["Water"].add_nuclide("H1", 2.0)
+    mats["Water"].add_element("O", 1.0)
+    mats["Water"].set_density("g/cm3", 1.0)
+    mats["Water"].add_s_alpha_beta("c_H_in_H2O")
+    mats["Water"].name = "Water"
+
+    p = pkg_resources.resource_filename(__name__, "dagmc_differentiate_mat.h5m")
```

Comment:
In the interest of minimizing the number of binary files in the repo and it's overall size, would one of the other existing DAGMC files in the regression tests work for this test instead?